makefile

```diff
@@ -36,8 +36,8 @@ else ifeq ($(strip $(COMPILER)),gfortran)
 	FOR = gfortran
 	FFLAGS = -cpp -std=gnu -fopenmp -march=native -ffree-line-length-512 -fcray-pointer -I$(OBJDIR) -J$(OBJDIR)
 
-	GCC_VERSION=$(shell gcc -v 2>&1 | grep -Po "^gcc version \K(\d+)")
-	ifeq ($(GCC_VERSION),10)
+	GCC_VERSION_GT_10 := $(shell expr `gcc -dumpversion | cut -f1 -d.` \>= 10)
+	ifeq "${GCC_VERSION_GT_10}" "1"
```

Comment:
Just with an eye on future changes, is it better to get the actual version number in a a variable, then compare for `>=10`? In case we need to compare to different versions at other points. Although at that point it may be worth rethinking the whole build system.